test/models/user/notifications/onboarding_community_notification_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+class User::Notifications::OnboardingCommunityNotificationTest < ActiveSupport::TestCase
+  test "keys are valid" do
+    user = create :user
+
+    notification = User::Notifications::OnboardingCommunityNotification.create!(user:)
+
+    assert_equal "#{user.id}|onboarding_community|", notification.uniqueness_key
+    assert_equal "Community is at the heart of Exercism. Try our forums and mentoring, get involved on YouTube and Twitch. Check the community section for more info!", notification.text # rubocop:disable Layout/LineLength
```

Comment:
It's probably better to actually use I18n here for less updating if we change the text. You'll need to check the actual I18n string. I just guess, but the syntax should be right.

```suggestion
    assert_equal I18n.t('notifications.onboarding_community'), notification.text
```

Reply:
Yeah I could do that. I normally don't like to do that though, as we're then basically duplicating application logic in the tests, which is a code smell to me. I'll happily update

Reply:
It's the pattern we have elsewhere so it's probably good to be consistent, and then maybe we should have seperate tests for the actual content. Ping me on Slack if you want to discuss though - I'm happy to change my mind.

Reply:
I'm fine with it